big-practice/src/js/text.js

```diff
@@ -0,0 +1,19 @@
+// Url API
+const accountApi = 'http://localhost:3000/accounts';
+const moviesApi = 'http://localhost:3000/movies';
+const EmptyText = '';
+
+// Messages
+const MESSAGES = {
```

Comment:
Can you use a format for variable name? I saw you're using camelCase, PascalCase and Uppercase in this file.